console/controllers/MyController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace console\controllers;
+
+use Yii;
+use console\models\Person;
+use yii\console\Controller;
+use yii\db\Command;
+use yii\helpers\ArrayHelper;
+
+class MyController extends Controller
+{
+ 
+   public function actionIndex()
+    {
+
+$z = 0;
```

Comment:
Please fix the codestyle. Here should be 4*2 spaces in the line $z = 0; 
in order to be inside the block. Use your IDE (phpstorm) for proper code formating